index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="style.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&display=swap"
+        rel="stylesheet">
+    <title>Mishelle Olivares Portfolio</title>
+</head>
+
+<body>
+    <header class="toolbar">
+        <a href="" class="logo">My Logo</a>
+        <nav class="navigation">
+            <img src="images/Icon-Menu.png" alt="menu_icon" id="icon-menu" class="hamburger">
+        </nav>
+    </header>
+    <main>
+        <section class="hero-section">
+
+            <h1 id="hello" class="container">Hey there. <br>
+                I’m Mishelle.</h1>
+            <h2 id="developer" class="container">I’m a software <br> developer</h2>
+            <p> I can help you to build a product,<br> feature or website Look through <br> some of my work and
+                experience!
+                <br> If you like what you see and have <br> a project you need coded, <br>
+                don’t hesitate to contact me.
+            </p>
+            <div class="socialMedia">
+                <ul class="social">
+                    <li><img src="images/twitter.png" alt="Twitter logo" class="social"></li>
+                    <li><img src="images/Linkedin.png" alt="Linkedin logo" class="social"></a></li>
+                    <li><img src="images/medium.png" alt="Medium" class="social"></li>
+                    <li></a><img src="images/github.png" alt="GitHub" class="social"></li>
+                    <li></a><img src="images/angellist.png" alt="Angellist" class="social"></a></li>
+                </ul>
```

Comment:
- [ ] [Optional]👍 Great to see that you have social icons in your project! However, I noticed that there is an issue with your code implementation. To ensure proper functionality and accessibility, I recommend updating your icons list structure as follows:

```html
<ul>
    <li>
       <a href="#">
          <img src="icon1.png" alt="Icon 1" /> 
        </a> 
    </li>
    <li>
       <a href="#">
          <img src="icon2.png" alt="Icon 2" /> 
        </a> 
    </li>
    <li>
       <a href="#">
          <img src="icon3.png" alt="Icon 3" /> 
        </a> 
    </li>
</ul>
```

In this structure, each icon image is wrapped in an `<a>` tag, allowing users to click on the icons and be directed to their respective social media profiles. The `<ul>` and `<li>` tags are used to create an ordered list for the icons.

Remember to replace the `href="#"` with the actual URLs of your social media profiles.

By implementing the updated structure, your social icons will be properly linked and enhance the user experience. Keep up the good work! 👍